src/stacktrace.cc

```diff
@@ -319,8 +319,8 @@ print_dump_stackmap_flamegraph(std::vector<std::pair<stack_trace_t, stack_info_t
 
 		const stack_trace_t &stack_trace = sorted_stack[i].first;
 
-		for (size_t j = info.stack_depth - 1; j >= 0; j--) {
-			print_backtrace_symbol_flamegraph(stack_trace[j], semicolon);
+		for (size_t j = 0; j < info.stack_depth; ++j) {
+			print_backtrace_symbol_flamegraph(stack_trace[info.stack_depth - j], semicolon);
```

Comment:
I think the index is a bit incorrect so it should be change as follows.
```suggestion
		for (size_t j = 0; j < info.stack_depth; ++j) {
			print_backtrace_symbol_flamegraph(stack_trace[info.stack_depth - j - 1], semicolon);
```
Let's say `info.stack_depth` is `5`, then the first argument of `print_backtrace_symbol_flamegraph` will be as follows.
- current: `4`, `3`, `2`, `1`, `0`
- with this change: `5`, `4`, `3`, `2`, `1`

Or this is also fine.
```suggestion
		for (size_t j = 1; j <= info.stack_depth; ++j) {
			print_backtrace_symbol_flamegraph(stack_trace[info.stack_depth - j], semicolon);
```

Reply:
Thanks, I wasn't careful.
The first one seems better for me, so I changed as you suggested.

Reply:
I actually like the second one, but no problem with the current change.